tests/system_test.py

```diff
@@ -488,6 +493,29 @@ def wait_ports(self, **retry_kwargs):
         wait_ports(self.ports_family, **retry_kwargs)
 
 
+class NginxServer(Process):
+    """
+    Sets up an ngix server
+    """
+    def __init__(self,
+                 config_path: str,  # Full path of templated config file (string) /blah-blah/nginx/nginx-template.conf
+                 env: dict[str, str],  # A dict of env variables which will be substituted in the config_path file.
+                 name: str="nginx",
+                 expect: int=Process.RUNNING,
```

Comment:
When the type is annotated, there is supposed to be spaces around the equals sign, https://peps.python.org/pep-3107/#syntax

```
def foo(a: expression, b: expression = 5):
    ...
```

I guess the check is disabled in the python checker ci job because there are already places that break these formatting conventions

Reply:
Done !